manim/mobject/svg/brace.py

```diff
@@ -101,14 +102,20 @@ def __init__(
         for mob in mobject, self:
             mob.rotate(angle, about_point=ORIGIN)
 
-    def put_at_tip(self, mob, use_next_to=True, **kwargs):
-        if use_next_to:
-            mob.next_to(self.get_tip(), np.round(self.get_direction()), **kwargs)
-        else:
-            mob.move_to(self.get_tip())
+    def put_at_tip(self, mob, rotate=False, **kwargs):
```

Comment:
Passing `use_next_to` makes this break, no? You might still want to make it available but as a deprecated parameter with a decorator `@deprecated_params`

Reply:
Hm, looking at it again, i understand what you mean. Maybe accept the parameter but not actually use them? Also, you can currently take a look at parametric surface to see deprecation of `u_min, u_max`